doc/classes/RichTextLabel.xml

```diff
@@ -341,14 +353,19 @@
 	</methods>
 	<members>
 		<member name="bbcode_enabled" type="bool" setter="set_use_bbcode" getter="is_using_bbcode">
+			Changes whether the label uses BBCode.
```

Comment:
Per the writing guidelines: http://docs.godotengine.org/en/latest/community/contributing/docs_writing_guidelines.html#use-if-true-to-describe-booleans

If [code]true[/code] the label uses BBCode formatting. Default value: [code]false[/code].